collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
Yes 'Any' 'Any' would show everything. I don't know if it is a valid use case or not. The ability exists right now. It wouldn't bother me to remove that combination if it can let us save a lot of time collecting issues.